freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaCapabilities.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.freeipa.client;
+
+import com.sequenceiq.freeipa.client.model.Config;
+
+import java.util.Set;
+
+public class FreeIpaCapabilities {
+
+    // users for password setting in FreeIPA Plugin
```

Comment:
the comment seems wrong. Maybe say "Attribute name to store users passwords ...."

Reply:
fixed